### PR TITLE
ChristianMDCH-001 chore: adding name file for roll call

### PR DESCRIPTION
**Pull request by ChristanMDCH**

### CODE REVIEW

File: ChristianMDCH/ChristianMDCH.md [hunk 1]
```diff
@@ -0,0 +1 @@
+ChristianMDCH Christian Hernandez
```
Comment (shammy24) [inline ChristianMDCH/ChristianMDCH.md:1]: The exercise is not completed
please review the code

**Review by shammy24 — COMMENTED** [review]
please look into the pull request

File: ChristianMDCH/ChristianMDCH.md (hunk 1 above)
Comment (ChristanMDCH) [reply ChristianMDCH/ChristianMDCH.md:1]: Code has been checked. Please review again

**Review by shammy24 — APPROVED** [review]
Looks good

thank you

**Pull request merged**